apps/upd/src/css/upd-custom.css

```diff
@@ -473,4 +473,12 @@ margin: 0px 20px 0px 20px;
 
 .chart-legend .legend-title-text {
   line-height: 18px !important;
+}
+
+.p-datatable.p-datatable-sm .p-datatable-header {
```

Comment:
Rather than defining new CSS rules, try applying pre-existing bootstrap classes whenever possible. [Here are the docs for Flex utilities for example](https://getbootstrap.com/docs/5.1/utilities/flex/#enable-flex-behaviors). But the same goes for font size and colour. If Bootstrap doesn't have what you need, you're probably still better off just defining a new class and adding it to the component's HTML. Not only is it more transparent, but then you can also re-use it.
This makes it easier to find rather than being hidden away in one of many CSS files, and helps ensure consistency